source/PluginDev/Assets/GooglePlayGames/Editor/template-AndroidManifest.txt

```diff
@@ -8,6 +8,9 @@
     
     <uses-sdk android:minSdkVersion="8" android:targetSdkVersion="16" /> 
  
+    <uses-permission android:name="android.permission.GET_ACCOUNTS"/>
```

Comment:
can we make these optionally added?  for example if the client id is not entered (or adda checkbox to the setup dialog?) then add these.  It seems heavy handed to make every game add these permissions.